Python/errors.c

```diff
@@ -510,6 +510,30 @@ _PyErr_GetExcInfo(PyThreadState *tstate,
     Py_XINCREF(*p_traceback);
 }
 
+PyObject*
+_PyErr_GetActiveException(PyThreadState *tstate)
+{
+    _PyErr_StackItem *exc_info = _PyErr_GetTopmostException(tstate);
+    PyObject *exc = exc_info->exc_value;
+    Py_XINCREF(exc);
+    return exc;
+}
+
+PyObject*
+PyErr_GetActiveException()
```

Comment:
If you don't put void, the function accepts anything:

```suggestion
PyErr_GetActiveException(void)
```